hcx-onboard/src/test/java/org/swasth/hcx/controllers/BaseSpec.java

```diff
@@ -330,4 +330,35 @@ protected String applicantVerifyOnboardVerificationTable(){
         String query = "INSERT INTO onboard_verification(participant_code,primary_email,primary_mobile,createdon,updatedon,expiry,phone_verified,email_verified,status,regenerate_count,last_regenerate_date,attempt_count, comments,phone_short_url,phone_long_url,onboard_validation_properties,participant_validation_properties) " + " VALUES('testhctes13.yopmail@swasth-hcx','testhctes13@yopmail.com','9620499129','169719173417','169719173417','2555824693417',false,true,'failed',0,'2023-10-12T13:37:12.533Z','5','','','','{\"email\": \"verification\",\"phone\": \"activation\"}',' {\"email\": \"verification\",\"phone\": \"activation\"}')";
         return query;
     }
+
+    protected String apiAccessSecretRequestBody() throws JsonProcessingException {
+        Map<String , Object> participant = new HashMap<>();
+        participant.put("email","testhcx@gmail.com");
+        participant.put("participant_code","payr_test1234@swasth-hcx");
+        return JSONUtils.serialize(participant);
+    }
+
+    protected String apiAccessSecretBody() throws JsonProcessingException {
+        Map<String , Object> participant = new HashMap<>();
+        participant.put("participant_code","payr_test1234@swasth-hcx");
+        return JSONUtils.serialize(participant);
+    }
+
+    protected String apiAccessSecretRequest() throws JsonProcessingException {
+        Map<String , Object> participant = new HashMap<>();
+        participant.put("user_id","hcxtest477@yopmail.com");
+        return JSONUtils.serialize(participant);
+    }
+
+    protected String apiAccessSecretRequestbdy() throws JsonProcessingException {
+        Map<String , Object> participant = new HashMap<>();
+        return JSONUtils.serialize(participant);
+    }
+
+    protected String apiAccessSecret() throws JsonProcessingException {
```

Comment:
Don't give combination of names, define the name in which context, you are using the request body. example: apiAccessSuccessRequestBody etc..